www/js/controllers.js

```diff
@@ -77,8 +76,36 @@ angular.module('starter.controllers', ['urlConstant'])
     }
   };
 
-  
+  $scope.trailMarkers = [];
+
+  var populateMarkers = function (markers, data) {
+    for (var i = 0; i < data.length; i++){
+      markers.push({
+        latitude: data[i].start_coordinates.latitude, 
+        longitude: data[i].start_coordinates.longitude,
+        id: data[i].id,
+        icon: data[i].icon
+      })
+    };
+    console.log(markers);
+  }
+
+  $http.get(apiUrl+'trails').success(function(data, status, xhr){
+    populateMarkers($scope.trailMarkers,data.trails)
+  });
+
+  $rootScope.$on('searchResults', function (param1, param2) {
+    // $scope.trailMarkers = $scope.results;
+    // console.log($scope.trailMarkers);
+    $scope.trailMarkers = [];
+    populateMarkers($scope.trailMarkers,param2.trails)
+  });
 
+  $scope.hihi = function() {
+    console.log("hihi");
```

Comment:
will be deleted.... its just for me to test the ng-click on markers which still hasn't been done